superset/db_engine_specs/oracle.py

```diff
@@ -58,3 +63,14 @@ def epoch_to_dttm(cls) -> str:
     @classmethod
     def epoch_ms_to_dttm(cls) -> str:
         return "TO_DATE('1970-01-01','YYYY-MM-DD')+(1/24/60/60/1000)*{col}"
+    
+    @classmethod
+    def fetch_data(cls, cursor: Any, limit: int) -> List[Tuple]:
```

Comment:
The signature of this method has changed as of #10707 making `limit: Optional[int] = None`

Reply:
OK，thank you very much.

Reply:
@villebro please help me check it.what is problem whit pre-commit?